RecoMuon/DetLayers/src/MuonDetLayerGeometry.cc

```diff
@@ -51,6 +54,26 @@ void MuonDetLayerGeometry::addCSCLayers(const pair<vector<DetLayer*>, vector<Det
   }    
 }    
 
+void MuonDetLayerGeometry::addGEMLayers(const pair<vector<DetLayer*>, vector<DetLayer*> >& gemlayers) {
+
+  for(auto const it : gemlayers.first) {
+    gemLayers_fw.push_back(it);
+    //    gemLayers_all.push_back(*it);
+    allForward.push_back(it);
+    //    allEndcap.push_back(*it);
+    //    allDetLayers.push_back(*it);
```

Comment:
please remove commented out lines, unless there is a really good reason to keep them here